src/pages/Governance/Proposal/card/VoteButtons.tsx

```diff
@@ -128,25 +163,49 @@ export default function VoteButtons({proposalId}: Props) {
           fullWidth
           size="large"
           variant="primary"
-          sx={{justifyContent: "start"}}
+          sx={{
+            justifyContent: "start",
+            backgroundColor: primaryColor,
+            "&:hover": {
+              backgroundColor: primaryColorOnHover,
+            },
+          }}
           startIcon={<CheckCircleOutlinedIcon />}
-          onClick={() => onVote(true)}
+          onClick={openVoteForModal}
         >
           {voteFor}
         </Button>
         <Button
           fullWidth
           size="large"
           variant="primary"
-          sx={{justifyContent: "start", backgroundColor: negativeColor}}
+          sx={{
+            justifyContent: "start",
+            backgroundColor: negativeColor,
+            "&:hover": {
+              backgroundColor: negativeColorOnHover,
+            },
+          }}
           startIcon={<CancelOutlinedIcon />}
-          onClick={() => onVote(false)}
+          onClick={openVoteAgainstModal}
         >
           {voteAgainst}
         </Button>
       </Stack>
       {voteOnSuccessSnackbar}
       {voteOnFailureSnackbar}
+      <ConfirmationModal
+        open={voteForModalIsOpen}
+        shouldPass={true}
+        onConfirm={() => onVote(true)}
+        onClose={closeVoteForModal}
+      />
+      <ConfirmationModal
```

Comment:
Is it possible for both modals to be open simultaneously?

Reply:
It's not. Because the FOR/AGAINST buttons will not be clickable once one of the modal is open.